ui/tests/pages/secrets/backend/database/connection.js

```diff
@@ -20,7 +19,7 @@ export default create({
   username: fillable('[data-test-input="username"]'),
   password: fillable('[data-test-input="password"]'),
   save: clickable('[data-test-secret-save]'),
-  addRole: clickable('[data-test-secret-create]'), // only from connection show
+  addRole: clickable('[data-test-add-role]'), // only from connection show
```

Comment:
I think we can remove the comment as this selector language indicates what it is saying.
```suggestion
  addRole: clickable('[data-test-add-role]'),
```